tests/system/test_active_response/test_netsh_windows_command/test_firewall_alerts/data/playbooks/generate_events.yaml

```diff
@@ -0,0 +1,55 @@
+- name: Clean alerts file
+  hosts: manager
+  tasks:
+
+    - name: Truncate alert.json
+      include_role:
+        name: manage_alerts
+        tasks_from: truncate_alert_json.yaml
```

Comment:
Move this to another playbook

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3787/commits/e4ded61433bf9f440dd1336be139e4ecb01581ec)